pkg/spec/deployment.go

```diff
@@ -134,6 +137,8 @@ func (deployment *DeploymentSpecMod) createKubernetesController() (*ext_v1beta1.
 	// TODO: merge with already existing labels and avoid duplication
 	deploymentSpec.Template.ObjectMeta.Labels = deployment.Labels
 
+	deploymentSpec.Template.ObjectMeta.Annotations = deployment.Annotations
```

Comment:
If you do https://github.com/kedgeproject/kedge/pull/454#discussion_r152549094 this is not needed!